deps/process.lua

```diff
@@ -124,6 +124,31 @@ local function exit(self, code)
   process.stderr:_end()
 end
 
+-- Returns the resident set size of the current process in bytes
+local function memoryUsage(self)
+  return uv.resident_set_memory()
```

Comment:
Shouldn't this return {rss = resident_set_memory} instead of just resident_set_memory?

Reply:
I simplified that return value since we aren't returning the other heap stats, but it could definitely still be a table for compatibility/future extensibility.